qa/suites/rgw/multifs/overrides.yaml

```diff
@@ -10,6 +10,7 @@ overrides:
         rgw crypt require ssl: false
         rgw sts key: abcdefghijklmnop
         rgw s3 auth use sts: true
+        rgw relaxed s3 bucket names: true
```

Comment:
enabling this on the s3-tests run causes several test cases to fail:
> FAILED s3tests_boto3/functional/test_s3.py::test_bucket_create_naming_bad_starts_nonalpha
FAILED s3tests_boto3/functional/test_s3.py::test_bucket_create_naming_dns_underscore
FAILED s3tests_boto3/functional/test_s3.py::test_bucket_create_naming_dns_dash_at_end
FAILED s3tests_boto3/functional/test_s3.py::test_bucket_create_naming_dns_dot_dot
FAILED s3tests_boto3/functional/test_s3.py::test_bucket_create_naming_dns_dot_dash
FAILED s3tests_boto3/functional/test_s3.py::test_bucket_create_naming_dns_dash_dot